superset/assets/src/SqlLab/actions/sqlLab.js

```diff
@@ -77,6 +81,26 @@ export function resetState() {
   return { type: RESET_STATE };
 }
 
+export function startQueryValidation(query) {
+  if (query == null) {
+    const msg = "programmer error: tried to validate a null query object";
+    console.error(msg);
```

Comment:
I think we have lint rules against console statements. Maybe just throw new Error(...) instead?

Reply:
Those functions can't be triggered w/ a null query param right now anyway, so I thought I'd match the rest of the code in that file and drop the extra assertion